scripts/zones/Bastok_Markets/npcs/Harmodios.lua

```diff
@@ -47,9 +47,9 @@ entity.onEventFinish = function(player, csid, option)
     if csid == 430 then
         player:setCharVar("WildcatBastok", utils.mask.setBit(player:getCharVar("WildcatBastok"), 10, true))
     elseif csid == 490 then
+        player:setCharVar("comebackQueenCS", 2)
         player:startEvent(491)
     elseif csid == 491 then
-        player:setCharVar("comebackQueenCS", 2)
     end
```

Comment:
This change leaves this block empty. (elseif csid == 491 then)
Should be deleted

Reply:
Guess could take out the entire thing then yea?